### PR TITLE
Samanthas updates 

### PR DESCRIPTION
**Pull request by Smoothie16**
Added my name to both files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# TatraTeam
\ No newline at end of file
+# TatraTeam
+## Samantha Bergdahl
```
Comment (CezarBudeci) [inline README.md:2]: Make it a list item

**Review by CezarBudeci — CHANGES_REQUESTED** [review]

**Review by jozefv-git — CHANGES_REQUESTED** [review]
Change paragraph to list item in the index

File: README.md (hunk 1 above)
Comment (Smoothie16) [reply README.md]: Updated

**Review by CezarBudeci — APPROVED** [review]
LGTM

**Review by jozefv-git — APPROVED** [review]
Finally

**Pull request merged**